tests/unittests/test_model.py

```diff
@@ -314,3 +314,67 @@ def test_change_unit_system(minimal_complete_model, unit_system, raises_before_v
                 minimal_complete_model.minimum_analysis_ply_thickness,
                 initial_minimum_analysis_ply_thickness * conversion_factor_by_us[unit_system.value],
             )
+
+
+def test_hdf5_composite_cae_export(
```

Comment:
I think it makes sense to parameterize this test with the options of `layup_representation_3d` and `offset_type`